src/feature/relay/dns.c

```diff
@@ -2026,6 +2033,26 @@ dns_launch_correctness_checks(void)
   }
 }
 
+/** If appropriate, if IPv6 DNS failed, reset the countner every 24 hours */
+static void
+dns_launch_ipv6_broken_reset(void)
```

Comment:
"launch" could mean now, or later.

Can we say "schedule" instead?

```suggestion
dns_schedule_ipv6_broken_reset(void)
```